<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Congratulation</class>
 <widget class="QDialog" name="Congratulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="BeltUpBtn_">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/resource/Belt_DownToUp.png</normaloff>:/img/resource/Belt_DownToUp.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="CenterUpBtn_">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/resource/Compose.png</normaloff>:/img/resource/Compose.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="CutterUpBtn_">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/resource/Cutter.png</normaloff>:/img/resource/Cutter.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>65</width>
     <height>65</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
